README.md

```diff
@@ -19,10 +19,11 @@ It records a spoken request from the connected microphones, sends it to the Goog
 ## Run the sample
 
 - Configure the Google API Console Project to use the [Google Assistant API][google-assistant-api-config].
-- Download the `client_secret_NNNN.json` file from the [credentials section of the Console][console-credentials].
+- Download the `client_secret_NNNN.json` file from the [credentials section of the Console][console-credentials]. Make sure to use "other" option and not "Android" when creating Oauth Client
```

Comment:
This is true. But we do include instructions on what client to use in the previous step.

Reply:
There are lot of stackoverflow posts where people have used the "Android" client (given this is **Android** things, common confusion)

Reply:
Have there been a lot of questions about this problem?

I would probably move your additional comment to a separate:

**Note:** line above the Download step to make sure the order is preserved.